src/julia.expmap

```diff
@@ -35,6 +35,9 @@
     __ZN4llvm23createLowerSimdLoopPassEv;
     LLVMExtra*;
 
+    /* Necessary for loader.c to find main() within repl.c */
+    main;
```

Comment:
We should rename this --- we can call it anything we want, and it's weird for a library to contain a `main` symbol.